eclipsesource/theia
# Support multi-selection of tree items in tree views

Signed-off-by: Thomas Mäder <t.s.maeder@gmail.com>


#### What it does
Adds multi-selection support to tree views contributed via VS Code API

Fixes #9074

Contributed on behalf of STMicroelectronics

#### How to test
The attached extension contributes a tree view that has multi-select enabled. There is a command "Print Selection" which prints out the selected items on the extension side. Make sure the behavior is consistent with VS Code.

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
